charts/internal/shoot-system-components/charts/csi-driver-manila/templates/daemonset-node.yaml

```diff
@@ -0,0 +1,229 @@
+            privileged: true
+            capabilities:
+              add: ["SYS_ADMIN"]
+            allowPrivilegeEscalation: true
+          image: {{ index $.Values.images "csi-driver-nfs" }}
+          args :
+            - "--v=2"
+            - "--nodeid=$(NODE_ID)"
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--drivername=gardener-{{ .protocolSelector | lower }}.nfs.csi.k8s.io"
+            - "--mount-permissions={{ .node.mountPermissions }}"
+          env:
+            - name: NODE_ID
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: CSI_ENDPOINT
+              value: unix:///csi/csi.sock
+          ports:
+            - containerPort: {{ .node.livenessProbe.healthPort2 }}
+              name: healthz
+              protocol: TCP
+          livenessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /healthz
+              port: healthz
+            initialDelaySeconds: 30
+            timeoutSeconds: 10
+            periodSeconds: 30
+{{- if $.Values.resources.driverNode }}
+          resources:
+{{ toYaml $.Values.resources.driverNode | indent 12 }}
+{{- end }}
+          volumeMounts:
+            - name: {{ .protocolSelector | lower }}-fwd-plugin-dir
+              mountPath: /csi
+            - name: pods-mount-dir
+              mountPath: /var/lib/kubelet/pods
+              mountPropagation: "Bidirectional"
+
+        - name: {{ .protocolSelector | lower }}-registrar
+          image: {{ index $.Values.images "csi-node-driver-registrar" }}
+          args:
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)
+            - -v=5
+          env:
+            - name: DRIVER_REG_SOCK_PATH
+              value: /var/lib/kubelet/plugins/{{ .protocolSelector | lower }}.manila.csi.openstack.org/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+{{- if $.Values.resources.nodeDriverRegistrar }}
+          resources:
+{{ toYaml $.Values.resources.nodeDriverRegistrar | indent 12 }}
+{{- end }}
+          volumeMounts:
+            - name: {{ .protocolSelector | lower }}-plugin-dir
+              mountPath: /csi
+            - name: registration-dir
+              mountPath: /registration
+
+        - name: {{ .protocolSelector | lower }}-manila-liveness-probe
+          image: {{ index $.Values.images "csi-liveness-probe" }}
+          args:
+            - --csi-address=/csi/csi.sock
+            - --health-port={{ .node.livenessProbe.healthPort }}
+{{- if $.Values.resources.livenessProbe }}
+          resources:
+{{ toYaml $.Values.resources.livenessProbe | indent 12 }}
+{{- end }}
+          volumeMounts:
+            - name: {{ .protocolSelector | lower }}-plugin-dir
+              mountPath: /csi
+
+        - name: {{ .protocolSelector | lower }}-nfs-liveness-probe
```

Comment:
Technically not necessary because manila forwards the requests to NFS [ref](https://github.com/kubernetes/cloud-provider-openstack/blob/829d3737ea77b6c21b743eae00473c375d88d366/pkg/csi/manila/identityserver.go#L41)